rest-api-spec/src/main/resources/rest-api-spec/test/indices.get_field_mapping/10_basic.yml

```diff
@@ -50,3 +50,20 @@ setup:
 
   - match: {test_index.mappings.text.mapping.text.type:     text}
   - match: {test_index.mappings.text.mapping.text.analyzer: default}
+
+---
+"Get field mapping with local is deprecated":
+
+  - skip:
+      features: ["warnings", "node_selector"]
+
+  - do:
+      node_selector:
+        version: "7.8.0 - "
+      warnings:
+        - "Use [local] in get field mapping requests is deprecated.The parameter will be removed in the next major version"
```

Comment:
I think a space would need to be added here, otherwise this test will fail.

Reply:
Good catch. It indeed failed for many of the CI tests ...